### PR TITLE
bring in changes from cedar

### PR DESCRIPTION
**Pull request by arehman199**

### CODE REVIEW

**Comment by arehman199** [comment]
@kergoth Kindly review.

File: meta-mel/conf/local.conf.append.cyclone5 [hunk 1]
```diff
@@ -60,3 +60,17 @@ MPV = "${@bb.utils.contains('INCLUDE_MPV', 'yes', 'mpv', '', d)}"
...
+# packages in your rootfs then set INCLUDE_COMMERCIAL_MULTIMEDIA to yes.
+
+INCLUDE_COMMERCIAL_MULTIMEDIA = "no"
+
+COMMERCIAL_MULTIMEDIA = "${@bb.utils.contains('INCLUDE_COMMERCIAL_MULTIMEDIA', 'yes', 'gstreamer1.0-plugins-ugly gstreamer1.0-libav', '', d)}"
+LIC_WHITELIST_COMMERCIAL = "${@bb.utils.contains('INCLUDE_COMMERCIAL_MULTIMEDIA', 'yes', ' commercial_gstreamer1.0-plugins-ugly \
+                                            commercial_gstreamer1.0-libav commercial_lame commercial_libmad commercial_mpeg2dec ', '', d)}"
+
+LICENSE_FLAGS_WHITELIST += "${LIC_WHITELIST_COMMERCIAL}"
+CORE_IMAGE_EXTRA_INSTALL_append += "${COMMERCIAL_MULTIMEDIA}"
```
Comment (kergoth) [inline meta-mel/conf/local.conf.append.cyclone5]: I never noticed, but the commit message of this commit is completely wrong. The prefix of the commit message summary indicates the file/recipe being modified. This modifies local.conf.append.cyclone5, not gstreamer.

**Comment by kergoth** [comment]
Merging from cedar periodically is a good thing. In this case, c98f7138ce02038a1f04ad4f695d2b3db27aeb3e needs to be reverted on master after the merge, as we don't use our custom mechanism for that there. Please add that revert to your branch and I'll see about merging.

**Comment by arehman199** [comment]
@kergoth Thanks for the review. I've added a commit to revert the change introduced in c98f713.

**Comment by kergoth** [comment]
I shouldn't have merged this, and you should have tested it:
```
ERROR: No recipes available for:
  /scratch/dogwood/archiver/meta-mentor/meta-mentor-staging/recipes-kernel/lttng/lttng-tools_2.6.0.bbappend
  /scratch/dogwood/archiver/meta-mentor/meta-mel/intel/recipes-graphics/mesa/mesa_10.6.3.bbappend
```

I'll fix it, but in the future, when doing merges, make sure it's possible to build.

**Comment by kergoth** [comment]
Looks like we also need to revert the addition of the 11.0 mesa, as 11.1 is in oe-core. Testing now.

**Pull request merged**